### PR TITLE
CHDEXCHAE-60 / Hidden resource flag for publishers

### PR DESCRIPTION
**Pull request by mutantsan**

### CODE REVIEW

**Review by ThrawnCA — CHANGES_REQUESTED** [review]
Failing lint check.

Are tests being run locally before pushing changes?

**Comment by mutantsan** [comment]
no

**Review by duttonw — APPROVED** [review]

**Comment by iaroslav13** [comment]
We are now running the linter locally. The issue has been fixed.

File: ckanext/resource_visibility/logic/action.py [hunk 1]
```diff
@@ -39,6 +36,19 @@ def package_show(next_func, context, data_dict):
     return data_dict
 
 
+def remove_hidden_resources(context, data_dict):
+    for res in data_dict["resources"]:
+        if _is_resource_visible(res, data_dict):
+            continue
```
Comment (ThrawnCA) [inline ckanext/resource_visibility/logic/action.py]: Since this is the only action within the loop, why not just make it "if not visible then set flag"?

**Review by ThrawnCA — APPROVED** [review]

File: ckanext/resource_visibility/logic/action.py (hunk 1 above)
Comment (mutantsan) [reply ckanext/resource_visibility/logic/action.py]: It's more convenient for me.

**Review by ThrawnCA — APPROVED** [review]

**Pull request merged**